app/src/main/res/layout-land/welcome.xml

```diff
@@ -11,44 +11,49 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:paddingStart="72dp"
+    android:paddingEnd="72dp">
 
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="0dp"
+    <com.google.android.flexbox.FlexboxLayout
+      android:id="@+id/flex_layout"
```

Comment:
We should be naming this based on what it represents in the UI, not what its data structure/widget type is. 'flex_layout' is really vague since it doesn't tell you anything about the layout or what it contains. Perhaps 'welcome_message_container' is clearer?

Reply:
Done